compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/DecomposeWinogradPass.cpp

```diff
@@ -82,6 +82,94 @@ struct FoldWinogradOpUnitDims : public OpRewritePattern<TransformOp> {
+/// `i = m + r - 1` and `r` is the output tile size.
+///
+/// The tile of the filter transform decomposes into two matrix multiplications:
+/// `matmul(G, matmul(tile(f), transpose(G)))`
+/// The matrix `G` is a precomputed constant.
+///
+/// The result of the decomposition will look like the following:
+/// ```
+/// %tf = iree_linalg_ext.winograd.filter_transform
+///     output_tile_size(6) kernel_size(3) kernel_dimensions([0, 1])
+///     ins(%in_tile : tensor<3x3xf32>)
+///     outs(%out_tile : tensor<8x8xf32>) -> tensor<8x8xf32>
+/// ```
+/// Decomposes to
+/// ```
+/// %G = arith.constant dense<[...]>
+/// %GT = arith.constant dense<[...]>
+/// %init_0 = linalg.fill ins(%zero) outs(%out_tile) -> tensor<3x8xf32>
+/// %mm_0 = linalg.matmul ins(%in_tile, %GT : tensor<3x3xf32>, tensor<3x8xf32>)
+///     outs(%init_0 : tensor<3x8xf32>) -> tensor<3x8xf32>
+/// %init_1 = linalg.fill ins(%zero) outs(%out_tile) -> tensor<8x8xf32>
+/// %mm_1 = linalg.matmul ins(%G, %mm_0 : tensor<8x3xf32>, tensor<3x8xf32>)
+///     outs(%init_1 : tensor<8x8xf32>) -> tensor<8x8xf32>
+/// ````
+struct DecomposeWinogradFilterTransform
+    : public OpRewritePattern<WinogradFilterTransformOp> {
+  using OpRewritePattern<WinogradFilterTransformOp>::OpRewritePattern;
+
+  LogicalResult matchAndRewrite(WinogradFilterTransformOp transformOp,
+                                PatternRewriter &rewriter) const override {
+    Location loc = transformOp.getLoc();
+    auto funcOp = transformOp->getParentOfType<mlir::FunctionOpInterface>();
+    if (!funcOp) {
+      return rewriter.notifyMatchFailure(transformOp,
+                                         "Could not find parent function");
+    }
+    rewriter.setInsertionPointToStart(&funcOp.getFunctionBody().front());
+
+    Value inputSlice = transformOp.input();
+    Value outputSlice = transformOp.output();
+    if (transformOp.getInputRank() != 2 || transformOp.getOutputRank() != 2) {
+      return rewriter.notifyMatchFailure(transformOp, "Winograd op not tiled");
+    }
+    const int64_t inputTileSize = transformOp.getInputTileSize();
+    const int64_t kernelSize = transformOp.getKernelSize();
+    ArrayRef<int64_t> kernelDims = transformOp.getKernelDimensions();
+    llvm::SmallSetVector<int64_t, 2> kernelDimsSet(kernelDims.begin(),
+                                                   kernelDims.end());
+    Type elementType = transformOp.getOutputType().getElementType();
+    Value zeroF32 = rewriter.create<arith::ConstantOp>(
+        loc, rewriter.getZeroAttr(elementType));
+    const float *GT{nullptr};
+    const float *G{nullptr};
+    GT = IREE::LinalgExt::Winograd::GT_6x6_3x3;
+    G = IREE::LinalgExt::Winograd::G_6x6_3x3;
+    Value GTV = IREE::LinalgExt::createValueFrom2DConstant(
+        GT, kernelSize, inputTileSize, loc, rewriter);
+    Value GV = IREE::LinalgExt::createValueFrom2DConstant(
+        G, inputTileSize, kernelSize, loc, rewriter);
```

Comment:
Can we pass `IREE::LinalgExt::Winograd::G.*_6x6_3x3` to `createValueFrom2DConstant` directly?